apis/v1alpha1/types.go

```diff
@@ -55,63 +46,21 @@ type EnhancedMetrics struct {
 	ShardLevelMetrics []*string `json:"shardLevelMetrics,omitempty"`
 }
 
-// The range of possible hash key values for the shard, which is a set of ordered
-// contiguous positive integers.
-type HashKeyRange struct {
-	EndingHashKey   *string `json:"endingHashKey,omitempty"`
-	StartingHashKey *string `json:"startingHashKey,omitempty"`
-}
```

Comment:
I'm wondering what caused all these types to disappear

Reply:
I regenerated with the latest version of code-gen... not sure :)